api_app/services/schema_service.py

```diff
@@ -58,26 +66,26 @@ def enrich_workspace_template(template) -> dict:
     """
     workspace_default_properties = read_schema('workspace.json')
     azure_ad_properties = read_schema('azuread.json')
-    return enrich_template(template, [workspace_default_properties, azure_ad_properties])
+    return enrich_template(template, [workspace_default_properties, azure_ad_properties], is_update=is_update)
 
 
-def enrich_workspace_service_template(template) -> dict:
+def enrich_workspace_service_template(template, is_update: bool = False) -> dict:
     """Adds to the provided template all UI and system properties
     Args:
         template: [Template to which UI and system properties are added].
```

Comment:
We have inconsistency with docstrings in the api code - some methods are documented and some aren't. As this method has a docstring already could do with adding the new is_update argument to it here